diff --git a/mkspecs/devices/linux-rasp-pi-g++/qmake.conf b/mkspecs/devices/linux-rasp-pi-g++/qmake.conf
index a6365d8..d6fea47 100644
--- a/mkspecs/devices/linux-rasp-pi-g++/qmake.conf
+++ b/mkspecs/devices/linux-rasp-pi-g++/qmake.conf
@@ -6,12 +6,14 @@ include(../common/linux_device_pre.conf)
 
 QT_QPA_DEFAULT_PLATFORM = wayland
 
-QMAKE_LFLAGS            += -Wl,-rpath-link,$$[QT_SYSROOT]/opt/vc/lib
+QMAKE_LFLAGS           += -Wl,-rpath-link,$$[QT_SYSROOT]/opt/vc/lib
 
 QMAKE_LIBDIR_OPENGL_ES2 = $$[QT_SYSROOT]/opt/vc/lib
 QMAKE_LIBDIR_EGL        = $$QMAKE_LIBDIR_OPENGL_ES2
 
-QMAKE_INCDIR_EGL        = $$[QT_SYSROOT]/opt/vc/include $$[QT_SYSROOT]/opt/vc/include/interface/vcos/pthreads
+QMAKE_INCDIR_EGL        = $$[QT_SYSROOT]/opt/vc/include \
+                          $$[QT_SYSROOT]/opt/vc/include/interface/vcos/pthreads \
+                          $$[QT_SYSROOT]/opt/vc/include/interface/vmcs_host/linux
 QMAKE_INCDIR_OPENGL_ES2 = $${QMAKE_INCDIR_EGL}
 
 QMAKE_LIBS_EGL          = -lEGL -lGLESv2
@@ -28,7 +30,7 @@ contains(DISTRO, squeeze) {
     DISTRO_OPTS += hard-float
 }
 
-QMAKE_CFLAGS            += \
+QMAKE_CFLAGS           += \
                           -marm \
                           -mfpu=vfp \
                           -mtune=arm1176jzf-s \
